README.md

```diff
@@ -70,6 +82,10 @@ Example Playbooks
       log_level: INFO
       apm_enabled: "true" # has to be set as a string
       logs_enabled: true  # log collection is available on agent 6
+      process_config: # on agent 6
+        enabled: "true" # to enable process-agent
+        scrub_args: true # to enable/disable process args scrubbing; default value is true
```

Comment:
Could be a bit clearer here, something like `enables scrubbing of sensitive arguments from the command line`